.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.2
```

Comment:
We should probably consider using ruby `2.4.0` if we can. What do you guys think?